### PR TITLE
Add changelog for 1.11.6

### PR DESCRIPTION
**Pull request by guillep2k**
As title

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,28 @@ This changelog goes through all the changes that have been made in each release
...
+  * Use media links for img in post-process (#10515) (#11504)
+  * Ensure public repositories in private organizations are visible and fix admin organizations list (#11465) (#11475)
+  * Set correct Content-Type value for Gogs/Gitea webhooks (#9504) (#10456) (#11461)
+  * Allow all members of private orgs to see public repos (#11442) (#11459)
+  * Whenever the ctx.Session is updated, release it to save it before sending the redirect (#11456) (#11457)
+  * Forcibly clean and destroy the session on logout (#11447) (#11451)
+  * Use session for retrieving org  teams (#11438) (#11439)
+  * Fix /api/v1/orgs/* endpoints by changing parameter to :org from :orgname (#11381)
+  * Add tracked time fix to doctor (part of #11111) (#11138)
+* ENHANCEMENT
```
Comment (techknowlogick) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,28 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix webpack chunk loading with STATIC_URL_PREFIX (#11526) (#11544)
+* BUILD
```
Comment (techknowlogick) [inline CHANGELOG.md]:
```suggestion
```

**Review by jolheiser — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,26 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.11.6](https://github.com/go-gitea/gitea/releases/tag/v1.11.6) - 2020-05-30
+
+* BUGFIXES
+  * Fix missing authorization check on pull for public repos of private/limited org (#11656) (#11683)
```
Comment (lafriks) [inline CHANGELOG.md:10]: This should be security

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,26 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix wrong milestone in webhook message (#11596) (#11612)
+  * Prevent (caught) panic on login (#11590) (#11598)
+  * Fix commit page js error (#11527)
+  * Use media links for img in post-process (#10515) (#11504)
+  * Ensure public repositories in private organizations are visible and fix admin organizations list (#11465) (#11475)
+  * Set correct Content-Type value for Gogs/Gitea webhooks (#9504) (#10456) (#11461)
+  * Allow all members of private orgs to see public repos (#11442) (#11459)
+  * Whenever the ctx.Session is updated, release it to save it before sending the redirect (#11456) (#11457)
+  * Forcibly clean and destroy the session on logout (#11447) (#11451)
+  * Use session for retrieving org  teams (#11438) (#11439)
```
Comment (lafriks) [inline CHANGELOG.md]: This also should be security

**Review by lafriks — CHANGES_REQUESTED** [review]

**Review by lafriks — APPROVED** [review]

**Comment by 6543** [comment]
Need Frontport to master and v1.12 branch

**Comment by guillep2k** [comment]
So, there's no "front-port" label. 😂

**Comment by lunny** [comment]
@guillep2k Now we have https://github.com/go-gitea/gitea/labels?q=frontport .

**Comment by 6543** [comment]
hihi frontport 🎉

**Pull request merged**